pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -103,35 +103,24 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
 			Name: "testdisk",
 		})
-		vmiBytes, _ := json.Marshal(&vmi)
```

Comment:
The commit message conveys that this commit does more than what is actually being done.